design/one-pager-resource-connectivity-mvp.md

```diff
@@ -638,7 +638,279 @@ parameters:
 
 ### Amazon Web Services
 
-TBD.
+In Amazon Web Services (AWS), `KubernetesCluster` and `MySQLInstance` claims are satisfied by `EKSCluster` and `RDSInstace` resource classes. 
+Similar to GCP, one network and one or more sub-networks are needed to connect `EKSCluster` and `RDSInstance` instances:
+
+* `AwsVpc`: creates a virtual private cloud (VPC) in AWS.
+* `AwsSubnet`: creates a virtual subnetwork within a `AwsVpc`
+
+However, unlike `GKECluster`, setting up an `EKSCluster` is less straightforward and requires more configurations. This is mostly because the worker nodes are not directly managed by the EKS cluster. Instead regular EC2 instances are launched and configured to communicate with the cluster. While creating new instances can be done at the time of cluster creation, a few network and security related resources need to be created previously:
+
+* `AwsSecurityGroup`: allows the cluster to communicate with worker nodes. It logically groups the resources that could communicate with each other within a VPC, and also adds ingress and egress traffic rules.
+* `AwsIamRole`: enables EKS to make calls to other AWS services to manage the resources.
+* `AwsIamRolePolicyAttachment`: attaches required policies the EKS role.
+* `AwsInternetGateway`: enables the nodes to have traffic to and from the internet. This is necessary because most workloads have a UI that needs to be accessed from the internet.
+* `AwsRouteTable`: for routing internet traffic from `AwsSubnet`s to `AwsInternetGateway`.
+* `AwsRoutTableAssociation`: associates an `AwsRouteTable` to a `AwsSubnet`.
+
+In addition, `RDSInstance`s also need following resources, so that they are accessible by the the worker nodes:
```

Comment:
Nitpick: `need the following` instead of `need following` would be more grammatically correct